max_obstacle_height: 0.60  # assume something like an arm is mounted on top of the robot
obstacle_range: 2.5
raytrace_range: 3.0
footprint: [[0.2425, 0.215], [0.2425, -0.215], [-0.2425, -0.215], [-0.2425, 0.215]]
inflation_radius: 0.50
update_frequency: 3.0
